RecoEgamma/EgammaTools/interface/HGCalShowerShapeHelper.h

```diff
@@ -84,48 +85,96 @@ class HGCalShowerShapeHelper {
           sigma2ww(0.0) {}
   };
 
-  HGCalShowerShapeHelper(edm::ConsumesCollector &&sumes);
```

Comment:
this file is out of sync with the master version in what looks like a missing code format.
Please update in full to avoid having someone dealing with conflicts in possible future backports.

Reply:
thanks Slava, not sure how that happened